projects/igniteui-angular/src/lib/core/styles/components/splitter/_splitter-theme.scss

```
@@ -79,6 +80,15 @@
 
     $variant: map.get($theme, '_meta', 'variant');
 
+    //remove unit from size
+    $size: math.div(map-get($theme, 'size'), map-get($theme, 'size') * 0 + 1);
```

Don't use old syntax like `map-get`. You already have the module `map` module imported.

```suggestion
    $size: math.div(map-get($theme, 'size'), map-get($theme, 'size') * 0 + 1);
```